### PR TITLE
initial Japanese localization

### PR DESCRIPTION
**Pull request by arubdesu**
All strings as of late 2024 AFAICT, spot-checked by Koji Arita.
May have issue on "Prevent future notifications for this application for " because sentence doesn't end that way grammatically in Japanese, trusting that `\n\n`'s & `<br />`'s will be interpreted correctly.

### CODE REVIEW

**Comment by russellhancox** [comment]
image

image

**Comment by russellhancox** [comment]
```----- Language ja -----
Missing: 4, ['Approve', 'Open...', 'authorize execution of ', 'authorize execution of the application ']
Extraneous: 0, []
Complete: 87.10%
```

3 of these missing strings are new and only apply to the new standalone mode but `Open...` is very important

**Comment by arubdesu** [comment]
Pardon, this should match en.lproj's strings now.
Please note again that grammatically 'authorize execution _space_' strings wouldn't work to have the app name follow afterwards (with the space boundary) as spaces are omitted in Japanese text, this is another place where an inline variable would/should be used

File: Source/gui/Resources/ja.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+"Santa is an application control system for macOS.\n\nThere are no user-configurable settings." = "SantaはmacOS向けアプリケーション制御システムです\n\n　ユーザーによる設定項目はありません";
+
+/* No comment provided by engineer. */
+"SHA-256" = "SHA-256";
+
+/* No comment provided by engineer. */
+"Signing ID" = "署名ID";
+
+/* The default message to show the user when an unknown application is blocked */
+"The following application has been blocked from executing<br />because its trustworthiness cannot be determined" = "次のアプリケーションは信頼性を判断できないため、<br />実行がブロックされました
```
Comment (russellhancox) [inline Source/gui/Resources/ja.lproj/Localizable.strings:78]: This string isn’t terminated

**Comment by russellhancox** [comment]
> Pardon, this should match en.lproj's strings now. Please note again that grammatically 'authorize execution _space_' strings wouldn't work to have the app name follow afterwards (with the space boundary) as e.g. the order of the phonetic word 'application' goes in the beginning/front of the sentence AND spaces are omitted in Japanese text, this is another place where an inline variable would/should be used

Interesting, I’ll have a look at updating the translations to include the app name tomorrow. Let’s get this in as-is for now and I’ll @ you when we can update to be more grammatically correct

**Review by russellhancox — CHANGES_REQUESTED** [review]

File: Source/gui/Resources/ja.lproj/Localizable.strings (hunk 1 above)
Comment (arubdesu) [reply Source/gui/Resources/ja.lproj/Localizable.strings]: fixicated, pardon

**Review by russellhancox — APPROVED** [review]

**Comment by russellhancox** [comment]
Thanks!

**Comment by arubdesu** [comment]
Just to reiterate, I mentioned in the previous comment where variables either aren't currently used in English or other localizations or wouldn't be appended to the end of the sentence for Japanese, please take into account that:
```"Prevent future notifications for this application for " = "このアプリケーションからの通知を停止します```
should have the variable of the time period be between このアプリケーションからの通知をand 停止します
Also "authorize execution of the application " should instead be pre-pended, so it would be for example "Slackアプリケーションの実行を許可する"

**Pull request merged**